src/BaGet/appsettings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "ApiKey": "",
+  "ApiKey": "SGD1953",
```

Comment:
Please revert this. I would recommend generating a new API key if this is what you use on your instance of BaGet